wordpress-mobile/WordPress-iOS
# Site settings: Missing writing settings

**Another part of** #7831 

**To test:**

On a .com and Jetpack site try editing the following settings:

- Date Format
- Time Format
- Start of the Week
- Posts per page

To verify that everything is OK you will have to use the API or wp-admin because Calypso is too aggressive about caching and doesn't consider other apps modifying the site settings.
